lib/facter/resolvers/windows/dmi_systemenclosure.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Facter
+  module Resolvers
+    class DMISystemEnclosure < BaseResolver
+      @log = Facter::Log.new(self)
```

Comment:
It's no longer recommended/needed to create a log instance, see 016811658a536f5c9c90d53e56fd7260f470bb8d Could you rebase on main and drop this line?